sysctl.d/50-pid-max.conf

```diff
@@ -11,6 +11,6 @@
 # (e.g. /etc/sysctl.d/90-override.conf), and put any assignments
 # there.
 
-# Bump the numeric PID range to its maximum of 2^22 (from the in-kernel default
-# of 2^16), to make PID collisions less likely.
+# Bump the numeric PID range to make PID collisions less likely.
+# 2^22 and 2^15 is typical maximum of 64b and 32b kernels respectively.
```

Comment:
I'd say `64bit` and `32bit` as it's clearer what it means

Reply:
i am pretty sure we should mention from which value we bump it.

Reply:
"typical"? why "typical"?

Reply:
It's in the commit message: "do not talk about default which may depend on nr_cpus." -- it would also be a maintenance burden to have the exact value there.

"Typical" because of `CONFIG_BASE_SMALL` in kernel (not sure if systemd would even run on such systems. It's a careful formulation not to give absolute expectations.

Reply:
s/typical/possible/

Reply:
CONFIG_BASE_SMALL really doesn't matter for systemd, I would not bother with that... I think there's little value with trying to be 100% correct in all possible cases. It's enought to be correct in the common cases for this.